Jenkinsfile_garbage_collector

```diff
@@ -0,0 +1,27 @@
+pipeline {
+    agent {
+    label "linux-arm64-docker"
+    }
+    stages {
+        stage('Garbage Collection') {
+            parallel {
+                stage('GC on Azure') {
+                    environment {
+                        PACKER_AZURE = credentials('packer-azure-serviceprincipal-sponsorship')
+                    }
+                    steps {
+                        catchError(buildResult: 'SUCCESS', stageResult: 'FAILURE') {
```

Comment:
```suggestion
```

This function is not needed anymore: we WANT the GC to fail bluntly in case of error